.gitignore

```diff
@@ -7,6 +7,7 @@
 
 # testing
 /coverage
+/supabase_data/bin/*
```

Comment:
Don't think we should be checking in the saved data... that would just duplicate data and bloat the repo.